README.org

```diff
@@ -233,6 +233,25 @@ This is great for performance, but means the data can become stale if you modify
 The =bibtex-actions-refresh= command will reload the caches, and you can call this manually.
 You can also call any of the =bibtex-actions= commands with a prefix argument: =C-u M-x bibtex-actions-insert-key=.
 
+Although not default =Bibtex-actions= also provides convenience functions for auto-refreshing cache when bib files change using filenotify. There are two such functions. =bibtex-actions-with-filenotify-local= and =bibtex-actions-filenotify-global=. The following snippet will refresh caches when one of the bib files change
+
+#+BEGIN_SRC emacs-lisp
+(bibtex-actions-with-filenotify-global #'bibtex-actions-refresh)
+
+(add-hook 'LaTeX-mode-hook (lambda () (bibtex-actions-with-filenotify-local #'bibtex-actions-refresh)))
+(add-hook 'org-mode-hook (lambda () (bibtex-actions-with-filenotify-local #'bibtex-actions-refresh)))
+#+END_SRC
+
+This will reload the cache whenever a bib file changes. A different option which might be useful is to merely invalidate the cache so that it is regenerated whenever a
+bibtex-actions command is called next. This can be achieved by the following code,
```

Comment:
```suggestion
This will reload the cache whenever a bib file changes. 
A different option which might be useful is to merely invalidate the cache so that it is regenerated whenever a bibtex-actions command is called next. 
This can be achieved by the following code:
```

Same, except why include this? I know we can, but it adds perhaps unnecessary complexity?

Reply:
I am fine with leaving it out. It is basically if someone doesn't like the cpu hit if they add a few entries in a row. But I am personally using the first one. My bib files don't change that often.

Reply:
WDYT about moving it to the wiki?

Reply:
Seems like a good idea

Reply:
I'm actually fine either way; just asking.

Reply:
Me too, I usually prefer it someone else makes organizational decisions:)